examples/frustum_pointnet/dataset_utils.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+NUM_HEADING_BIN = 12
+NUM_SIZE_CLUSTER = 8
+NUM_OBJECT_POINT = 512
+
+type2class = {'Car': 0, 'Van': 1, 'Truck': 2, 'Pedestrian': 3,
+              'Person_sitting': 4, 'Cyclist': 5, 'Tram': 6, 'Misc': 7}
+class2type = {type2class[t]: t for t in type2class}
+type2onehotclass = {'Car': 0, 'Pedestrian': 1, 'Cyclist': 2}
+type_mean_size = {'Car': np.array([3.88311640418, 1.62856739989, 1.52563191]),
```

Comment:
`type_mean_size` doesn't seem to reflect the proper variable name. Maybe something like `class_to_mean`?